cli/post-install.js

```diff
@@ -1,98 +1,64 @@
 import { executeCommand } from "./utils.js";
```

Comment:
Since now it's a build artifact - we can hide it from diff. Maybe even move it to `/dist` or `/lib`.

Reply:
@volovyks This ties in with [this comment](https://github.com/near/near-sdk-js/pull/261#discussion_r993457093).